### PR TITLE
Update README

### LINKED ISSUE #1451: Update README
### Description

Since we do not publish lisk-service-framework to the npm registry, the README should reflect the same

### Acceptance Criteria

- Remove irrelevant information from [README](https://github.com/LiskHQ/lisk-service/tree/development/framework#installation)

### PR DESCRIPTION
**Pull request by sameersubudhi**
### What was the problem?
This PR resolves #1451 

### How was it solved?
- [x] Update README with proper instructions to install `lisk-service-framework` as a dependency

### How was it tested?
- Executed the command locally

### CODE REVIEW

File: framework/README.md [hunk 1]
```diff
@@ -12,8 +12,9 @@ All modules can be easily replaced by other libraries when needed.
 
 ## Installation
 
+Set the target URL to the desired tarball distribution permalink.
 ```
-npm install lisk-service-framework
+npm install https://github.com/LiskHQ/lisk-service/raw/2fbc65f49ae5c5aaa3689667051dbd4082b8c25a/framework/dist/lisk-service-framework-1.4.10.tgz
```
Comment (ManuGowda) [inline framework/README.md:17]: is this how we currently doing it?

Comment (sameersubudhi) [reply framework/README.md:17]: Yes!

**Review by ManuGowda — APPROVED** [review]

File: framework/README.md (hunk 1 above)
Comment (ManuGowda) [reply framework/README.md:17]: Probably every-time we update the version we have remember to update this version i guess. 
we need to find a better approach, probably if we need to publish the `lisk-service-framework` we should consider doing that in this release?

Comment (sameersubudhi) [reply framework/README.md:17]: Everytime we make changes to the framework, we package the distribution and check it in the repo itself under the path: `./framework/dist`

**Pull request merged**